scripts/core/editor3/directive.js

```diff
@@ -111,6 +115,14 @@ class Editor3Directive {
 
         const store = createStore(this);
 
+        $scope.$watch('vm.value', (newValue, oldValue) => {
```

Comment:
When I started to work on it it was already using the editor, I just added the highlights. The editor is readonly.